commands/soup.js

```diff
@@ -0,0 +1,20 @@
+const talkedRecently = new Set();
+module.exports = {
+    name: 'soup',
+    description: "soup",
+    async run (client, message, args) {
+        if(message.channel.id == "500326922509615105")
+        {
```

Comment:
What is this, C#? Braces on newlines 🤢

Reply:
I don't know JS and honestly, I just got used to doing this in c#